### DISCUSSION TITLE
Cannot Create New Deta Base using NestJS.

### DISCUSSION THREAD

**Discussion by DKB0512** [deta/cloud-docs#449] [Help] [answered]
So I am trying to create a Deta Base with the NestJS code : 

```
async getHello() {
    const deta = Deta('KEY_HERE');
    const db = deta.Base('simple_db');
    await db.put({name: "Test", surname: "Testing"});
 }
```

But when I try to see the visor, it is giving me 304. How can I fix this? Am I doing something wrong here?

BTW I have used this tutorial for creating the NestJS project on deta : https://dev.to/adrianub/como-desplegar-nestjs-en-detash-2c14

**Comment by abdelhai** [comment, marked as answer]
try `deta logs --follow` to debug the issue. feel free to remove the `'KEY_HERE'` as project keys get automatically loaded into the deta sdk inside the micro.

**Reply by DKB0512** [reply]
I somehow managed to get it working. I tried to put that in async/await along with then() with console.log to see if it was working. Thanks.
